Unresolved reference: BuildConfig

Fresh checkout
`gradle build` results in 

```
kotlin-ir-plugin-template\kotlin-ir-plugin-native\src\main\kotlin\com\bnorm\template\TemplateCommandLineProcessor.kt: (36, 35): Unresolved reference: BuildConfig

FAILURE: Build failed with an exception.
```

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -3,10 +3,10 @@ buildscript {
 }
 
 plugins {
-  kotlin("jvm") version "1.4.20" apply false
-  id("org.jetbrains.dokka") version "0.10.0" apply false
-  id("com.gradle.plugin-publish") version "0.11.0" apply false
-  id("com.github.gmazzo.buildconfig") version "2.0.2" apply false
+  kotlin("jvm") version "1.4.30" apply false
+  id("org.jetbrains.dokka") version "0.10.1" apply false
+  id("com.gradle.plugin-publish") version "0.12.0" apply false
+  id("com.github.gmazzo.buildconfig") version "2.1.0" apply false
 }
 
 allprojects {

--- a/gradle/wrapper/gradle-wrapper.properties
+++ b/gradle/wrapper/gradle-wrapper.properties
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.7.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.2-bin.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists

--- a/kotlin-ir-plugin/build.gradle.kts
+++ b/kotlin-ir-plugin/build.gradle.kts
@@ -14,7 +14,7 @@ dependencies {
 
   testImplementation(kotlin("test-junit"))
   testImplementation("org.jetbrains.kotlin:kotlin-compiler-embeddable")
-  testImplementation("com.github.tschuchortdev:kotlin-compile-testing:1.2.6")
+  testImplementation("com.github.tschuchortdev:kotlin-compile-testing:1.3.4")
 }
 
 buildConfig {

